packages/experience-builder-sdk/src/core/editor/designComponentUtils.ts

```diff
@@ -102,8 +116,10 @@ export const resolveDesignComponent = ({
     },
     nodeId: node.data.id,
     parentId: node.parentId,
-    experienceDataSource: {},
-    experienceUnboundValues: componentFields.unboundValues,
+    designComponentDataSource: {},
+    designComponentUnboundValues: componentFields.unboundValues,
+    componentInstanceProps: node.data.props,
```

Comment:
Here we mean the top-level container props yeah?

Reply:
Yep root level design component props